codechain/__version__.py

```diff
@@ -0,0 +1,7 @@
+__title__ = "codechain"
+__description__ = "Python HTTP for Humans."
+__url__ = "https://codechain.io/"
+__version__ = "1.0.0"

```

Comment:
Leave it as "0.0.1" or "0.0.0" until things are settled down.

Reply:
I fixed it.